api/src/main/resources/schema/workflow.json

```diff
@@ -1,5 +1,5 @@
 {
-  "$id": "https://wg-serverless.org/workflow.schema.json",
+  "$id": "classpath:schema/workflow.schema.json",
```

Comment:
the new library tries to download the schema from a not existing url (Because wg-serverless-org is fake), hence changing to real location (which is library classpath)

Reply:
tell me if im wrong you already define the hard-coded path to the schema in code, if so then why here? i think it was by design to put some id like this and not use classpath